src/python/gem5/components/boards/se_binary_workload.py

```diff
@@ -111,3 +112,37 @@ def set_se_binary_workload(
 
         # Set whether to exit on work items for the se_workload
         self.exit_on_work_items = exit_on_work_items
+
+    def set_se_looppoint_workload(
```

Comment:
I think we changed the way we're doing simpoints. Can you follow what's done here; https://gem5-review.googlesource.com/c/public/gem5/+/64432